spanner/spanner_snippets/snippet.go

```diff
@@ -1820,11 +1822,27 @@ func queryWithHistory(ctx context.Context, w io.Writer, client *spanner.Client)
 }
 
 func createClients(ctx context.Context, db string) (*database.DatabaseAdminClient, *spanner.Client) {
-	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	// [START spanner_create_admin_client_for_emulator]
```

Comment:
Would prefer if we migrating the Spanner samples to the new Go sample style (each sample in a different file and the region tags including the import block), but all good to do this for now.